io-engine/src/core/env.rs

```diff
@@ -75,10 +80,10 @@ fn parse_mb(src: &str) -> Result<i32, String> {
 
 #[derive(Debug, Clone, StructOpt)]
 #[structopt(
-    name = package_description!(),
-    about = "Containerized Attached Storage (CAS) for k8s",
-    version = version_info_str!(),
-    setting(structopt::clap::AppSettings::ColoredHelp)
+name = package_description ! (),
```

Comment:
The previous seemed better formatted?

Reply:
Reverting back, had used IDE formatting.